packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -375,7 +375,7 @@ describe("ReadAttributeRequestItemProcessor", function () {
                 id: requestId,
                 createdAt: CoreDate.utc(),
                 isOwn: false,
-                peer: peer,
+                peer: sender,
```

Comment:
...You can keep the declaration `const sender = CoreAddress.from("Sender");` in order to use it for having an appropriate `sender` for the Request sent.

Reply:
For simplicity, I'll keep one variable `peer` here.